bindings/java/src/org/sleuthkit/datamodel/IngestStatusType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.datamodel;
+
+enum IngestStatusType {
```

Comment:
Let's make this IngestJobStatusType for clarity and make it an inner class of IngestJobInfo.